src/Components/Publishing/Sections/ImageSetPreview.tsx

```diff
@@ -192,12 +195,14 @@ const SubTitle = styled.div`
   display: flex;
 `
 const SubTitlePrompt = styled.div`
-  ${unica("s14", "medium")} ${pMedia.xs`
+  ${unica("s14", "medium")};
+  ${pMedia.xs`
     ${unica("s12", "medium")}
   `};
 `
 const SubTitleCount = styled.div`
-  ${unica("s14")} margin-left: 20px;
+  ${unica("s14")};
+  margin-left: 20px;
```

Comment:
@orta - did stylelint catch these things?

Reply:
Yes

Reply:
The error is more like "there is two rules on the same line" kinda thing